LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I wonder. Should we not add a caveat that the license of the models inside this module might differ from MIT?